tern/__main__.py

```diff
@@ -145,11 +145,11 @@ def main():
                         help="Change default working directory to specified"
                         " absolute path.")
     parser.add_argument('-dr', '--driver', metavar="DRIVER_OPTION",
-                        help="Required when running Tern in a container."
-                        "Using 'fuse' will enable the fuse-overlayfs driver "
-                        "to mount the diff layers of the container. If no "
-                        "input is provided, 'fuse' will be used as the "
-                        "default option.")
+                        default='default',
+                        help="Choose from the following storage drivers: \n"
+                        "overlay2: Use the kernel's overlay2 storage driver\n"
+                        "fuse: Use the fuse-overlayfs system tool\n"
+                        "If no option is given, 'default' will be used.")
```

Comment:
what is "default" in this case?

Reply:
The bulk copy while taking care of whiteouts method.

Reply:
Yeah, I meant that we might want to clarify this. A new user likely won't know what the "default" method is.

Reply:
Do you have a recommendation on what to call this method?

Reply:
I think default is fine but maybe phrasing like "If no option is given, the default bulk copy method will be used" ? just something that gives a hint as to what the default method is. If they want to know more, we have that info in the README.